src/platform/silabs/wifi/wf200/WifiInterfaceImpl.cpp

```diff
@@ -603,6 +588,16 @@ CHIP_ERROR WifiInterfaceImpl::InitWiFiStack()
     return CHIP_NO_ERROR;
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_IPV4
+void WifiInterfaceImpl::GotIPv4Address(uint32_t ip)
+{
+    ChipLogDetail(DeviceLayer, "DHCP IP=%ld.%ld.%ld.%ld", (ip & 0xFF), (ip >> 8 & 0xFF), (ip >> 16 & 0xFF), (ip >> 24 & 0xFF));
+    sta_ip = ip;
+
+    WifiInterfaceImpl::GetInstance().NotifyIPv4Change(true);
```

Comment:
If it is a class function, it shouldn't need to call the NotifyIPv4Change function with the GetInstance().

Is there a reason for this function?

Reply:
removed the GetInstance